packages/@ember/object/index.js

```diff
@@ -3,31 +3,103 @@ import { assert } from '@ember/debug';
 import { assign } from '@ember/polyfills';
 
 /**
-  Decorator that turns the target function into an Action
-  Adds an `actions` object to the target object and creates a passthrough
-  function that calls the original. This means the function still exists
-  on the original object, and can be used directly.
+  Decorator that turns the target function into an Action which can be accessed
+  directly by reference.
 
   ```js
-  export default class ActionDemoComponent extends Component {
+  export default class Tooltip extends Component {
```

Comment:
May want to add the

```js
import Component from '@ember/component';
import { set } from '@ember/object';
```

imports, since we'd probably expect `isShowing` in a glimmer component to be tracked, and thus not need `set`